experiments/hyperparameter_search/launch_sweep.py

```diff
@@ -0,0 +1,174 @@
+import argparse
+import random
+import yaml
+import os
+from dataclasses import dataclass
+
+from concurrent.futures import ProcessPoolExecutor
+
+import wandb
+import numpy as np
+import mo_gymnasium as mo_gym
+from mo_gymnasium.utils import MORecordEpisodeStatistics
+
+from morl_baselines.common.utils import reset_wandb_env
+
+from morl_baselines.common.experiments import ALGOS, ENVS_WITH_KNOWN_PARETO_FRONT, StoreDict
+
+@dataclass
+class WorkerInitData:
+    sweep_id: str
+    seed: int
+    config: dict
+    worker_num: int
+
+@dataclass
+class WorkerDoneData:
+    hypervolume: float
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--algo", type=str, help="Name of the algorithm to run", choices=ALGOS.keys(), required=True)
+    parser.add_argument("--env-id", type=str, help="MO-Gymnasium id of the environment to run", required=True)
+    parser.add_argument(
+        "--ref-point", type=float, nargs="+", help="Reference point to use for the hypervolume calculation", required=True
+    )
+
+    parser.add_argument("--wandb-entity", type=str, help="Wandb entity to use for the sweep", required=False)
+    parser.add_argument("--project-name", type=str, help="Project name to use for the sweep", default="MORL-Baselines")
+
+    parser.add_argument("--sweep-count", type=int, help="Number of times to run the sweep", default=10)
+    parser.add_argument("--num-seeds", type=int, help="Number of seeds to use for the sweep", default=3)
+
+    parser.add_argument(
+        "--train-hyperparams",
+        type=str,
+        nargs="+",
+        action=StoreDict,
+        help="Override hyperparameters to use for the train method algorithm. Example: --train-hyperparams num_eval_weights_for_front:10 timesteps_per_iter:10000",
+        default={},
+    )
+
+    args = parser.parse_args()
+
+    parser.add_argument("--config-name", type=str, help="Name of the config to use for the sweep.", default=f"{args.algo}.yaml")
+
+    args = parser.parse_args()
+
+    return args
+
+def train(worker_data: WorkerInitData) -> WorkerDoneData:
+    # Reset the wandb environment variables
+    reset_wandb_env()
+
+    seed = worker_data.seed
+    group = worker_data.sweep_id
```

Comment:
Is the group parameter really necessary?

Reply:
Not really, but it allows the runs related to the sweep to be grouped in Runs in Workspace.